kubernetes/apps/observability/loki/app/helmrelease.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     mediaType: application/vnd.cncf.helm.chart.content.v1.tar+gzip
     operation: copy
   ref:
-    tag: 6.29.0
+    tag: 6.30.0
```

Comment:
This is a cool bot. Did you make it? Also what kind of homelab do you run? I found your GH from a PR on the binance connector and saw home-ops and had to poke around :)

Cheers!

Reply:
Oh, thanks!>.. I run 1 nuc and 2 chromboxes with Talos, a NAS.
I didn't make the bot... I'm mostly participating of the home-operations community..
https://discord.com/invite/home-operations and following what @onedr0p and @bjw-s has been doing.. 
https://github.com/onedr0p/cluster-template

See you in the discord :)

Reply:
@blackjid https://github.com/home-operations/charts-mirror/releases/tag/2025.5.4

Reply:
thanks :)